beacon_node/lighthouse_network/src/types/topics.rs

```diff
@@ -126,7 +126,7 @@ pub fn all_topics_at_fork<E: EthSpec>(fork: ForkName, spec: &ChainSpec) -> Vec<G
         enable_light_client_server: true,
         subscribe_all_subnets: true,
         subscribe_all_data_column_subnets: true,
-        sampling_subnets: &sampling_subnets,
+        sampling_subnets: sampling_subnets,
```

Comment:
Use shorthand initializer

Reply:
Addressed in https://github.com/sigp/lighthouse/pull/7578/commits/1adcc4a5406911fa8677a51507a787a9c1565e2c